gm4_zauber_cauldrons/data/gm4_zauber_cauldrons/functions/player/wormhole_targeting/set_dimension.mcfunction

```diff
@@ -16,4 +16,4 @@ tp @a[tag=gm4_zc_wormhole_consumer,limit=1] ~.5 ~.7 ~.5
 # play arrival animation
 particle minecraft:portal ~.5 ~1.3 ~.5 .25 .25 .25 0 100 force @a[distance=..32]
 execute positioned ~.5 ~.7 ~.5 run playsound minecraft:entity.enderman.teleport player @a[distance=..8] ~ ~ ~ 1 .3
-execute unless entity @e[type=area_effect_cloud,tag=gm4_zauber_cauldron,dx=0,dy=0,dz=0] run summon tnt ~.5 ~.7 ~.5
+execute unless entity @a[tag=gm4_zc_wormhole_consumer,tag=gm4_ta_jammed,limit=1] unless entity @e[type=area_effect_cloud,tag=gm4_zauber_cauldron,dx=0,dy=0,dz=0] run summon tnt ~.5 ~.7 ~.5
```

Comment:
I'd much rather see a `gm4_zc_prevent_explosion` tag, not specific to TA

Reply:
This has been changed to a score check for ZC. I was going to implement it as a tag, but then modules would try to remove the tag and there's no way to verify that another module still wants the tag set